### ISSUE TITLE
Challenge console output

### ISSUE THREAD

**Issue by QuincyLarson** [FreeCodeCamp/FreeCodeCamp#5097]
Thanks to Thad for reporting. His words:

I have a question about the code output area. I'm referring to the area just under the Reset, Help and Bug buttons on the left side of the page at FCC. It seems that there was a change to how that functions in the past few days.

In the past, when it was dictated that an array should show up in the code output area, it was displayed just like an array...["Split", "me", "into", "an", "array"]. Now, since the change a few days ago, it displays like...Split, me, into, an, array. I've attached a screenshot.

Is this the desired output or is it a bug? If it's the desired output, it doesn't seem very useful. If the answer is supposed to be an array, it's reasonable to expect to see an array (not just the array items) displayed there.

![](https://www.evernote.com/l/AqTp5MJdC1FPm4W_ujoPAbfIIoloZ80h4tgB/image.png)

**Comment by thadseaver** [comment]
Thanks @QuincyLarson

**Comment by shl-kfy** [comment]
hi, i have a similar one
bonfire, http://www.freecodecamp.com/challenges/bonfire-dna-pairing
my code is 
```
function pair(str) {
  var ans = new Array(str.length);
  for (var i = 0; i < str.length; i++) {
    if (str[i]==='C') ans[i]=["C","G"];
    if (str[i]==='G') ans[i]=["G","C"];
    if (str[i]==='T') ans[i]=["T","A"];
    if (str[i]==='A') ans[i]=["A","T"];
  }
    console.log(ans);
 return ans;
}

pair("GCG");

```
the result shows G,C,C,G,G,C but not [["G", "C"],["C", "G"],["G","C“]], is it a bug?

**Comment by cowboy63** [comment]
I have the same issue even if I add the formatting myself the code does not pass the test cases.

**Comment by ltegman** [comment]
@cowboy63 The console output formatting definitely needs to be fixed since it is misleading, but it doesn't impact the test cases. So correct code will still pass just fine.

**Comment by ltegman** [comment]
Looks like the culprit of this is in `end.js`. We're implicitly converting code output to a string but that uses the default `toString` method, which doesn't always format things the way we'd like.

**Comment by cowboy63** [comment]
Thank you ,  @ltegman

**Comment by BerkeleyTrue** [comment]
Closed by #6095
